services/api/src/lib/i18n-adapter.ts

```diff
@@ -0,0 +1,30 @@
+import { DEFAULT_LOCALE } from '@algomart/schemas'
+import i18next, { TFunction } from 'i18next'
+import Backend from 'i18next-fs-backend'
+import path from 'node:path'
+
+export default class I18nAdapter {
+  constructor() {
+    if (i18next.isInitialized) return
+    i18next.use(Backend).init({
+      fallbackLng: DEFAULT_LOCALE,
+      lng: DEFAULT_LOCALE,
+      ns: ['emails'],
+      backend: {
+        loadPath: path.join(__dirname, '../../locales/{{lng}}/{{ns}}.json'),
+        addPath: path.join(
+          __dirname,
+          '../../locales/{{lng}}/{{ns}}.missing.json'
+        ),
+      },
+    })
```

Comment:
Set this up to more closely align with the i18next documentation and also avoid calling init multiple times.